docker/faucet/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM debian:buster-20200803@sha256:a44ab0cca6cd9411032d180bc396f19bc98f71972d2398d50460145cab81c5ab AS toolchain
+
+# To use http/https proxy while building, use:
+# docker build --build-arg https_proxy=http://fwdproxy:8080 --build-arg http_proxy=http://fwdproxy:8080
+
+RUN apt-get update && apt-get install -y cmake curl clang git
+
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain none
+ENV PATH "$PATH:/root/.cargo/bin"
+
+WORKDIR /libra
+COPY rust-toolchain /libra/rust-toolchain
+RUN rustup install $(cat rust-toolchain)
+
+COPY cargo-toolchain /libra/cargo-toolchain
+RUN rustup install $(cat cargo-toolchain)
+
+FROM toolchain AS builder
+
+ARG ENABLE_FAILPOINTS
+COPY . /libra
+
+RUN ./docker/build-common.sh
+
+### Production Image ###
+FROM debian:buster-20200803@sha256:a44ab0cca6cd9411032d180bc396f19bc98f71972d2398d50460145cab81c5ab AS prod
+
+RUN mkdir -p /opt/libra/bin /opt/libra/etc
+
+COPY --from=builder /libra/target/release/libra-faucet /opt/libra/bin
```

Comment:
We don't need a new image, just add this `COPY` line to the existing `docker/mint/Dockerfile`.